### PR TITLE
edited zooma report to include annotamogram annotations for SCEA - 

### PR DESCRIPTION
**Pull request by ngeorgeebi**
not including study or bioentity fields as these anatomogram terms aren't currently linked to any datasets

### CODE REVIEW

File: zoomage_report.CURATED.tsv [hunk 1]
```diff
@@ -55815,3 +55815,81 @@ sampling site	macula lutea	http://purl.obolibrary.org/obo/UBERON_0000053	E-GEOD-
...
 sampling site	sinoatrial node	http://purl.obolibrary.org/obo/UBERON_0002351	E-HCAD-16	SAMN12046033	Nancy George	13/08/2020 16:00
 specimen with known storage state	fresh	http://purl.obolibrary.org/obo/OBI_0000971	E-HCAD-7	group2	Nancy George	13/08/2020 16:00
+organism part	airway submucosal gland	http://www.ebi.ac.uk/efo/EFO_0010666			Nancy George	28/08/2020 16:00
+organism part	alveolar sac	http://purl.obolibrary.org/obo/UBERON_0002169			Nancy George	28/08/2020 16:00
+organism part	body of pancreas	http://purl.obolibrary.org/obo/UBERON_0001150			Nancy George	28/08/2020 16:00
+organism part	central vein of liver	http://purl.obolibrary.org/obo/UBERON_0006841			Nancy George	28/08/2020 16:00
+cell type	central venous hepatic sinusoidal endothelial cell	http://www.ebi.ac.uk/efo/EFO_0010706			Nancy George	28/08/2020 16:00
+organism part	chorionic plate	http://purl.obolibrary.org/obo/UBERON_0004027			Nancy George	28/08/2020 16:00
+organism part	chorionic villus	http://purl.obolibrary.org/obo/UBERON_0007106			Nancy George	28/08/2020 16:00
+cell type	Clara cell	http://purl.obolibrary.org/obo/CL_0000158			Nancy George	28/08/2020 16:00
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:55825]: This should be "club cell", no?

File: zoomage_report.CURATED.tsv [hunk 2]
```diff
@@ -55815,3 +55815,81 @@ sampling site	macula lutea	http://purl.obolibrary.org/obo/UBERON_0000053	E-GEOD-
...
+organism part	collecting duct of renal tubule	http://purl.obolibrary.org/obo/UBERON_0001232			Nancy George	28/08/2020 16:00
+organism part	common bile duct	http://purl.obolibrary.org/obo/UBERON_0001174			Nancy George	28/08/2020 16:00
+organism part	cytotrophoblast	http://purl.obolibrary.org/obo/UBERON_0000319			Nancy George	28/08/2020 16:00
+organism part	decidua basalis	http://purl.obolibrary.org/obo/UBERON_0000453			Nancy George	28/08/2020 16:00
+organism part	decidual stromal cell	http://purl.obolibrary.org/obo/CL_2000002			Nancy George	28/08/2020 16:00
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:55830]: Category should be "cell type".

File: zoomage_report.CURATED.tsv [hunk 3]
```diff
@@ -55815,3 +55815,81 @@ sampling site	macula lutea	http://purl.obolibrary.org/obo/UBERON_0000053	E-GEOD-
...
+organism part	descending thin limb	http://purl.obolibrary.org/obo/UBERON_0005096			Nancy George	28/08/2020 16:00
+organism part	distal convoluted tubule	http://purl.obolibrary.org/obo/UBERON_0001292			Nancy George	28/08/2020 16:00
+organism part	distal tubule	http://purl.obolibrary.org/obo/UBERON_0004135			Nancy George	28/08/2020 16:00
+organism part	endometrial glandular epithelial cell	http://www.ebi.ac.uk/efo/EFO_0010710			Nancy George	28/08/2020 16:00
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:55834]: Category "cell type".

**Review by anjaf — APPROVED** [review]
Looks good. Just 3 small changes. And maybe you can add to the description what organs are these terms covering.
I'm approving now so you can merge when you've made the update.

File: zoomage_report.CURATED.tsv (hunk 1 above)
Comment (anjaf) [reply zoomage_report.CURATED.tsv:55825]: I see it has it as synonym. So should be ok.

Comment (ngeorgeebi) [reply zoomage_report.CURATED.tsv:55825]: changed it anyway 👍

**Pull request merged**